easybuild/easyconfigs/m/MetaGeneMark-2/MetaGeneMark-2-1.06-GCC-13.2.0.eb

```
@@ -0,0 +1,58 @@
+easyblock = 'MakeCp'
+
+name = 'MetaGeneMark-2'
+version = '1.06'
+
+homepage = 'https://github.com/gatech-genemark/MetaGeneMark-2'
+description = """MetaGeneMark-2 is an unsupervised metagenomic gene finder. 
+It improves on MetaGeneMark by adding models for better gene start prediction,
+as well as automatic selection of genetic code (4 or 11). The models for gene
+start prediction are based in part on the work done for GeneMarkS-2; they include
+Shine-Dalgarno RBS, non-Shine-Dalgarno (or non-canonical) RBS, and bacterial and
+archaeal promoter models (for use in cases of leaderless transcription).
+"""
+
+toolchain = {'name': 'GCC', 'version': '13.2.0'}
+
+sources = [{
+    'filename': '%(name)s-%(version)s.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/gatech-genemark',
+        'repo_name': '%(name)s',
+        'commit': 'a455a8b48ebc6f681b81202c0760d7830f015fe5',
+    }
+}]
+patches = ['MetaGeneMark-2-fix_Makefile.patch']
+checksums = [
+    {'MetaGeneMark-2-1.06.tar.gz': '9565a8174485b0fcad6a369551464abd5dfbff1770ed8aba2b872eedfa4b8374'},
+    {'MetaGeneMark-2-fix_Makefile.patch': '6f0fe0fefc053833f394e6ce47fe17596fda552a34e21901fd24d1f60a2d10d0'},
+]
+
+dependencies = [
+    ('Perl', '5.38.0'),
+    ('Boost', '1.83.0'),
+]
+
+build_cmd = ' && '.join([
+    'cd src',
+    'make -f Makefile.ubuntu'
+])
+
+parallel = 4
+
+fix_perl_shebang_for = ['bin/run_mgm.pl']
+
+files_to_copy = [
+    (['src/gmhmmp2', 'src/run_mgm.pl'], 'bin'),
+    (['src/*.h', 'src/*.hpp'], 'include'),
+    (['src/mgm2_11.mod', 'src/mgm2_4.mod'], 'lib'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/gmhmmp2'],
+    'dirs': ['include']
+}
+
+sanity_check_commands = ['perl bin/run_mgm.pl | grep seq']
```

```suggestion
sanity_check_commands = ['perl %(installdir)s/bin/run_mgm.pl | grep seq']
```

These are not ran from the install directory